### ISSUE TITLE
Admin My Profile - Ensure all controls are keyboard operable

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#245]
<img width="1680" alt="Screen Shot 2020-04-23 at 8 05 33 AM" src="https://user-images.githubusercontent.com/31868510/80097625-95bc5800-8539-11ea-80f3-f3b656efb003.png">

There are several controls missing from the tab order and out of reach of keyboard control.

Ensure that all controls are keyboard operable. You can read more about this success criterion here:

https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html
